soc/riscv/riscv-ite/it8xxx2/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 zephyr_sources(
 	soc.c
 )
+zephyr_library_sources_ifndef(CONFIG_RISCV_MUL_ISA __arithmetic.S)
```

Comment:
Keep in mind that `CONFIG_RISCV_MUL_ISA` will changed to `CONFIG_RISCV_ISA_EXT_M` when #45582 is merged.

Reply:
Ack. Thanks for the info.